### PR TITLE
isis: add cisco configurations

### PR DESCRIPTION
**Pull request by rubensfig**

### CODE REVIEW

File: network_configuration/isis.md [hunk 1]
```diff
@@ -189,3 +189,27 @@ using the two in-between routers switch-1 and switch-2 as hops.
 
 For all possible configuration options within isisd in FRR, please refer to the
 official FRR documentation [here](http://docs.frrouting.org/en/latest/isisd.html)
+
+## Cisco Interoperability
+
+When connecting the BISDN Switch and FRR with ISIS, it is essential to correctly configure
```
Comment (jklare) [inline network_configuration/isis.md:195]: I think we should add links to the docs regarding the default settings and how to change those for cisco and frr here.

Comment (rubensfig) [reply network_configuration/isis.md:195]: The attached Cisco config is meant to illustrate how to configure the metric-style.

FRR documentation PR is waiting for merge, https://github.com/FRRouting/frr/pull/8566. I could not pinpoint the default assignment in the code.

Comment (jklare) [reply network_configuration/isis.md:195]: The linked ISIS cisco config docs does not include any obvious hints regarding metric-style configuration for me. Could you point me to the relevant section?

Comment (jklare) [inline network_configuration/isis.md:195]:
```suggestion
When connecting Cisco routers and switches running BISDN Linux and FRR with ISIS, it is essential to correctly configure
```

File: network_configuration/isis.md [hunk 2]
```diff
@@ -189,3 +189,27 @@ using the two in-between routers switch-1 and switch-2 as hops.
...
+the `metric` setting. Per default, the Cisco Routers have the `metric narrow` configuration,
```
Comment (jklare) [inline network_configuration/isis.md:196]: is it "the `metric`setting" or "the `metric-style` setting" or "the metric-style" or "matching metric styles"

Comment (jklare) [inline network_configuration/isis.md:196]: I was not able to find the Cisco metric-style default you mention here. Please point me to the relevant docs section where you found it or change this to "Some Cisco Routers use `metric narrow` in their configuration" (or something similar)

File: network_configuration/isis.md [hunk 3]
```diff
@@ -189,3 +189,27 @@ using the two in-between routers switch-1 and switch-2 as hops.
...
 For all possible configuration options within isisd in FRR, please refer to the
 official FRR documentation [here](http://docs.frrouting.org/en/latest/isisd.html)
+
+## Cisco Interoperability
+
+When connecting Cisco routers and switches running BISDN Linux and FRR with ISIS, it is essential to correctly configure
+the `metric` setting. Per default, the Cisco Routers have the `metric-style narrow` configuration,
+while FRR chooses the wide metrics. Refer to [CISCO docs](https://www.cisco.com/c/en/us/support/docs/ip/integrated-intermediate-system-to-intermediate-system-is-is/13795-is-is-ip-config.html) for further documentation on configuring Cisco routers with ISIS. For specific information on `metric-syle` configurations, refer to [CISCO](https://www.cisco.com/c/en/us/td/docs/ios-xml/ios/iproute_isis/command/irs-cr-book/irs-l1.html#wp1681001735) and [FRR](http://docs.frrouting.org/en/latest/isisd.html#clicmd-metric-style[narrow|transition|wide]).
+
+Below is a sample configuration to connect the BISDN Linux FRR to a Cisco Router.
```
Comment (jklare) [inline network_configuration/isis.md:199]: where is this configuration applied? in which context can it be used? how does the corresponding configuration on the other side look?
I think without the context of this configuration, just pointing to the related docs is cleaner and less confusing.

**Comment by jklare** [comment]
lgtm

**Pull request merged**